lib/govuk_publishing_components/components/related_navigation_helper.rb

```diff
@@ -0,0 +1,245 @@
+class RelatedNavigationHelper
```

Comment:
:+1: This is a great refactor. Having all the helper logic in one place is much clearer.